src/cpio/client_providers/kms_client_provider/azure/azure_kms_client_provider_utils.cc

```diff
@@ -57,49 +57,49 @@ bool AzureKmsClientProviderUtils::isPrivate(
 /**
  * @brief Generate hex hash on wrapping key
  */
-std::string AzureKmsClientProviderUtils::CreateHexHashOnKey(
-    std::shared_ptr<EvpPkeyWrapper> publicKey) {
+absl::StatusOr<std::string> AzureKmsClientProviderUtils::CreateHexHashOnKey(
+    std::shared_ptr<EvpPkeyWrapper> public_key) {
   ERR_clear_error();
-  CHECK(isPrivate(publicKey) == 0)
+  CHECK(isPrivate(public_key) == 0)
       << "CreateHexHashOnKey only supports public keys";
 
   // Create a BIO to hold the public key in PEM format
-  BIOWrapper bioWrapper;
-  int result = PEM_write_bio_PUBKEY(bioWrapper.get(), publicKey->get());
+  BIOWrapper bio_wrapper;
+  int result = PEM_write_bio_PUBKEY(bio_wrapper.get(), public_key->get());
   if (result != 1) {
-    char errBuffer[MAX_OPENSSL_ERROR_STRING_LEN];
+    char err_buffer[MAX_OPENSSL_ERROR_STRING_LEN];
     char* error_string =
```

Comment:
from a consistency point of view we should make error_string const. We have many of these.

Reply:
If you look at the RHS of  this line `char* error_string = ERR_error_string_n(ERR_get_error(), error_string, sizeof(err_buffer));`, `error_string` is passed as `char*`. So I think it's correct to be non-const variable.
If you add const, it fails to compile.

If the it's a variable with long lifetime, maybe it's worth to write  `cosnt char* error_string = ERR_error_string_n(ERR_get_error(), const_cast<char*>error_string, sizeof(err_buffer));` to prevent change after the value assignment, but it only has a very short life. I think we can leave it.